JitsiConference.js

```diff
@@ -3495,7 +3504,7 @@ JitsiConference.prototype._shouldBeInP2PMode = function() {
     const peers = this.getParticipants();
     const peerCount = peers.length;
     const hasBotPeer = peers.find(p => p.getBotType() === 'poltergeist' || p.hasFeature(FEATURE_JIGASI)) !== undefined;
-    const shouldBeInP2P = peerCount === 1 && !hasBotPeer && !this._hasVisitors;
+    const shouldBeInP2P = peerCount === 1 && !hasBotPeer && !this._hasVisitors && !this.getMetadataHandler().getMetadata()?.recording?.isTranscribingEnabled;
```

Comment:
Why not just check the instance variable you added rather than check the metadata?